src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -430,6 +455,27 @@ impl A320Hydraulic {
         self.yellow_loop.is_pressurised()
     }
 
+    #[allow(clippy::too_many_arguments)]
+    // Update with same refresh rate as the sim
+    fn update_max_fixed_step(&mut self, context: &UpdateContext) {
+        while let Some(cur_time_step) = self.physics_updater.next() {
```

Comment:
**thought**: The `MaxStepLoop`'s name differs from the `max_fixed_step` name of this function. I wonder if those names should be the same?

Reply:
That's more a MaxFixedStepLoop maybe :)